.github/workflows/deploy.yml

```diff
@@ -179,3 +179,21 @@ jobs:
         with:
           message: |
             Preview url: ${{ steps.url_preview.outputs.NETLIFY_PREVIEW_URL }}
+
+  deploy_backend:
+    name: Deploy backend to AWS
+    runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request' && github.ref_name == 'main'
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up SSH key
+        run: |
+          mkdir -p ~/.ssh
+          echo "${{ secrets.CATCOLAB_NEXT_DEPLOYUSER_KEY }}" > ~/.ssh/id_ed25519
+          chmod 600 ~/.ssh/id_ed25519
+          ssh-keyscan ec2-18-217-205-18.us-east-2.compute.amazonaws.com >> ~/.ssh/known_hosts
```

Comment:
Would it still work if we replace this line with

```sh
ssh-keyscan backend-next.catcolab.org >> ~/.ssh/known_hosts
```

Reply:
yup! I pushed the changes